tests/Mf2/ClassicMicroformatsTest.php

```diff
@@ -238,4 +238,444 @@ public function testRelBookmarkUrl() {
 
 	}
 
+
+	/**
+	 * @see http://microformats.org/wiki/microformats2-parsing-issues#uf2_children_on_backcompat_properties
+	 */
+	public function testMf2ChildrenOnBackcompatProperties()
```

Comment:
It sounds like the resolution is that in this test, the `adr` should not be added as a `type` of the `h-card`. This will require some changes to the parser, and we also need a test here to check that `type` correctly has only one value.